core/templates/pages/topic-editor-page/modal-templates/questions-opportunities-select-difficulty-modal.component.html

```diff
@@ -40,4 +50,14 @@ <h3 *ngIf="linkedSkillsWithDifficulty.length > 0">
     display: block;
     font-size: 1em;
   }
+  @media screen and (max-width: 500px) {
+    .oppia-modal-header {
+      font-size: 22px;
+      /* !important is needed to override the margin enforced by h3 */
+      margin-bottom: 0 !important;
```

Comment:
I think this needs more explanation.

Reply:
I have removed the !important tag by using negative margin on parent div (instead of setting margin to 0 here)